### PR TITLE
added main README and project workflow docs

### PR DESCRIPTION
**Pull request by Skidle**
No issue.

Went a bit overboard with the project docs, if you think we don't need all of them, let me know.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
+# Sturdy Lamp 💡
```
Comment (Skidle) [inline README.md:1]: tip: you can click on `View file` for `.md` files and see the preview of a document

image

image

File: README.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+- [Overview](#overview)
+- [Features](#features)
+- [Technologies](#technologies)
+- [Local setup](#local-setup)
+- [Project workflow](#project-workflow)
+- [Authors](#authors)
+
+## Overview
+
+**What is it?** A forum where users can post their opinions.
```
Comment (Skidle) [inline README.md:12]: don't know what else to call it 🙈

File: README.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+**Why?** This project serves as a learning playground to deepen authors' knowledge in web technologies.
+
+**Demo:** [link]()
```
Comment (Skidle) [inline README.md:16]: will add it once we figure out deployment

File: docs/features/post-management.md
Comment (Skidle) [inline docs/features/post-management.md:1]: placeholders for this issue https://github.com/metisova/sturdy-lamp/issues/12, maybe it's better to remove them for now actually

File: docs/workflow/code-review.md [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+1. [Pick an issue to work on](./issue-management.md#how-to-pick-an-issue)
+
+2. Check out a new branch from the latest `main` branch
+    ```bash
+    git checkout -b [branch-name]
+    ```
+
+3. Solve the issue
+
+    > If you've made a lot of changes (>300 LOC), split the changes into multiple PRs - TBA how
```
Comment (Skidle) [inline docs/workflow/code-review.md:17]: you can either split the changes manually, use `git cherry-pick` if you were smart and making smaller commits, or do some magic with `git stash` and `git apply`. I general it's a good idea to commit and push often, this way you can check if you're over the limit already.

File: docs/workflow/code-review.md [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+
+4. Push the changes and open a PR
+
+    4.1. Add `Closes [issue-link]` to the description
```
Comment (Skidle) [inline docs/workflow/code-review.md:21]: that's a github feature on how to link PRs to issues https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword

File: docs/workflow/code-review.md [hunk 6]
```diff
@@ -0,0 +1,103 @@
...
+
+    4.3. Request a review and assign labels if necessary
+
+5. Create a PR
+
+    > *You can also give yourself a review, sometimes it helps to view all the changes one more time in Github.*
+
+6. Wait for a review
+
+    > If the reviewer doesn't respond in 1-3 days, then ping them until they give you a review. Getting a review is the author's responsibility.
```
Comment (Skidle) [inline docs/workflow/code-review.md:39]: `1-3 days` is pretty random, we can specify it better if needed

File: docs/workflow/code-review.md [hunk 7]
```diff
@@ -0,0 +1,103 @@
...
+
+    6.2. Do we need a test for this functionality?
+
+    6.3. Should we add a comment in the code to explain it?
+
+7. Once you add all the comments, click `Review changes` button:
+
+    7.1. If you don't have major objections to the code, then choose `Approve` and submit review
+
+    7.2. In all other cases choose `Request changes`
```
Comment (Skidle) [inline docs/workflow/code-review.md:95]: I don't know if this part actually makes sense, these guidelines are pretty generic, feel free to disagree, we can remove it.

File: docs/workflow/issue-management.md [hunk 8]
```diff
@@ -0,0 +1,61 @@
...
+    2.4. `Acceptance criteria` - issue outcomes, what have we achieved?
+
+    2.5. `Learning outcome` - how did we benefit from it personally, what did we learn?
+
+3. Add relevant labels:
+
+    3.1. Estimate how long the issue is going to take (1h / 2h / 3h)
+
+        If it takes more than 3 hours, then it should be split (e.g. into research and actual implementation)
+
```
Comment (Skidle) [inline docs/workflow/issue-management.md:27]: I also realized that we can estimate not only issues but whole "milestones" which are like collections of issues, based on our weekly availability. We should try and divide the whole project into milestones ahead of time then.

File: README.md [hunk 9]
```diff
@@ -0,0 +1,73 @@
...
+
+```bash
+# clone repository
+git clone git@github.com:metisova/sturdy-lamp.git
+```
+
+### Start the server
+
+```bash
+# cd into server
```
Comment (metisova) [inline README.md:42]: @Skidle test review

**Review by metisova — COMMENTED** [review]
All good :)

File: README.md [hunk 10]
```diff
@@ -0,0 +1,73 @@
...
+# install server's dependencies
+npm install
+
+# run server
+npm run start:server
+```
+
+### Start the client
+
+```bash
```
Comment (metisova) [inline README.md:54]: @Skidle test 2

**Review by metisova — COMMENTED** [review]
test

File: README.md (hunk 2 above)
Comment (metisova) [reply README.md:12]: @Skidle We can just write a simple text instead: "Sturdy Lamp is a forum where users can freely express their opinions. This project was created as a learning playground to help authors deepen their knowledge in web technologies."

or 

'**Project description:**A forum where users can post their opinions.
**Project purpose:** This project serves as a learning playground to deepen authors' knowledge in web technologies."

File: docs/workflow/code-review.md (hunk 4 above)
Comment (metisova) [reply docs/workflow/code-review.md:17]: @Skidle can we pls add it to the manual as well?

**Review by metisova — APPROVED** [review]

File: docs/workflow/code-review.md (hunk 4 above)
Comment (Skidle) [reply docs/workflow/code-review.md:17]: will do in a follow-up https://github.com/metisova/sturdy-lamp/issues/26

File: README.md (hunk 2 above)
Comment (Skidle) [reply README.md:12]: will do in a follow-up https://github.com/metisova/sturdy-lamp/issues/26

**Pull request merged**